README.md

```diff
@@ -111,6 +111,22 @@ Related Modules
 
 * [jsonld.js][]: An implementation of the [JSON-LD][] specification.
 
+### Using with React Native
```

Comment:
Looks like this might be better in the `Installation` section?

Reply:
The `Installation` section typically has to do with installing the /npm/ package, the `import` statements have traditionally been in `Usage`

Reply:
Ok.  It's under `Related Modules` now.  How about moving it up a few lines into the `Examples` section?  (Closest thing to `Usage` here.)

Reply:
@davidlehn - Thanks, I moved it up to Examples.